addons/bus/static/src/longpolling_bus.js

```diff
@@ -20,7 +20,7 @@ export class Longpolling extends EventBus {
 
         // PROPERTIES
         this._isActive = null;
-        this._id = _.uniqueId('bus');
+        this._id = multi_tab.currentTabId;
```

Comment:
It is not clear to me why this change is related to the fix at hand or why it is needed at all. What am I missing ?

Reply:
It was a mistake from https://github.com/odoo/odoo/commit/e2aeb5fc23cc0460a3edef7abf17442a8d2d414e breaking a little more the feature.
It is assumed the ids in `channels` are matching the ids in `lastPresenceByTab` (previously named `peers`)
It was no longer true since `lastPresenceByTab` is managed by `multi_tab` service (with it's own id).
With those non-matching ids, channels from other tabs were always considered outdated and therefore cleaned up.

It's not possible to make this test pass without this fix. And I think writing a test for this change/fix only would be basically writing the same test (which requires the entire fix to pass :p)

Reply:
Ok I understand, thanks for the clarification. Could you update the commit message to include the other changes that were required for the behavior to properly work ? That is the `unload` fix an the id change. I think those are the only two "additions" to the original commit, right ? Personally, I would have split the changes of the last commit in three commits, honestly. The original one that was to be forward ported, then the unload fix, then the tab id fix. If it is not too complicated at this point to filter out the changes into different commits, I think it would really help anyone who might one day look at those changes and try to understand what was going on. Thanks !